src/Illuminate/Database/Migrations/DatabaseMigrationRepository.php

```diff
@@ -178,4 +178,28 @@ public function setSource($name)
 		$this->connection = $name;
 	}
 
+	/**
+	 * Get migration entries from the database by path.
+	 *
+	 * @param  string  $path
+	 * @return array
+	 */
+	public function getByPath($path)
```

Comment:
This method doesn't make sense at the bottom of the file after all the getter methods. Put it somewhere else more suitable please.